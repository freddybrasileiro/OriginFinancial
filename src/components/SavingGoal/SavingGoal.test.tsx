import * as React from 'react';
import { shallow } from 'enzyme';
import { SavingGoal } from './index';

describe('SavingGoal', () => {
  it('Should render the SavingGoal component', () => {
    expect(
      shallow(<SavingGoal goal="Buy a house" width="xs" />)
    ).toMatchSnapshot();
  });
});
