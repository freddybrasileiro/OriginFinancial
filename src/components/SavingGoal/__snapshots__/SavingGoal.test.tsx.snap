// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SavingGoal Should render the SavingGoal component 1`] = `ShallowWrapper {}`;
